packages/utils/src/document/document-query.ts

```diff
@@ -0,0 +1,57 @@
+import dotenv from "dotenv";
+dotenv.config();
+
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const patientId = getEnvVarOrFail("PATIENT_ID");
+const facilityId = getEnvVarOrFail("FACILITY_ID");
+
```

Comment:
why is this script here? is it a leftover?

Reply:
testing script. That was what I was told to test that sdk worked.